libmscore/edit.cpp

```diff
@@ -4878,17 +4878,45 @@ void Score::undoAddCR(ChordRest* cr, Measure* measure, const Fraction& tick)
 
       Tuplet* t = cr->tuplet();
 
+      // For linked staves the length of staffList is always > 1 since the list contains the staff itself too!
+      const bool linked = ostaff->staffList().length() > 1;
+
       for (const Staff* staff : ostaff->staffList()) {
+
```

Comment:
Still an empty line too much IMHO

Reply:
Removed only one line. Now the other is gone too.